noxfile.py

```diff
@@ -40,6 +40,7 @@ def tests(session: nox.Session) -> None:
         "--cov-report=xml",
         "--durations=10",
         "--numprocesses=logical",
+        "--dist=loadgroup",
```

Comment:
[F] All tests that use the `tmpfile` flavour of the database (i.e. the tests that use parallel/distributed processing) have been grouped onto the same worker.